sentinel/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine:3.15 AS base_build
+
+RUN apk add --update --no-cache \
+  build-base \
+  curl \
+  nodejs~=16 \
+  npm~=8 \
+  tzdata \
+  libxslt \
+  bash \
+  jq    \
+  git
```

Comment:
Again, not sure why we'd want these...

Reply:
I guess jq is useful for debugging. If you exec inside the container and need to query stuff.